s3/main.tf

```diff
@@ -79,6 +79,15 @@ resource "aws_s3_bucket" "bucket" {
     }
   }
 
+  dynamic "lifecycle_rule" {
+    for_each = var.abort_incomplete_uploads == true ? ["include_block"] : []
+    content {
+      id                                     = "abort-incomplete-uploads"
+      enabled                                = true
+      abort_incomplete_multipart_upload_days = 7
```

Comment:
This sounds good to me. Could you double-check it with David if you haven't already? He ought to know we'll be deleting incomplete data.

Reply:
no problem here. thank you